<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>droneTypes</class>
 <widget class="QWidget" name="droneTypes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1794</width>
    <height>1500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 11pt &quot;Ubuntu Condensed&quot;;
background-color: rgb(193, 125, 17);
font: 13pt &quot;Ubuntu&quot;;</string>
   </property>
   <property name="text">
    <string>       Vehicle SetUp</string>
   </property>
  </widget>
  <widget class="QLabel" name="quadrator">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>391</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="rover">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>70</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="unknown_drone_ui">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>70</y>
     <width>381</width>
     <height>261</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="plane">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>420</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(136, 138, 133);
border-color: rgb(52, 101, 164);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tricopter_y">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>420</y>
     <width>411</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="unknown_drone_ui_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>420</y>
     <width>391</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="uav_main_comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>320</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ugv_combo_ui">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>320</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>320</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_plane_ui">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>690</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_tricopter">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>690</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_6">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>690</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(233, 185, 110);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Summary</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Firmware</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>AirFrame</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Radio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fligth Modes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Power</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Safety</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Tuning</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>560</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
